apps.yaml

```diff
@@ -220,3 +220,15 @@ HierCVAE:
         - inverse-design
         - ml-metadata
         - wp11
+tomato:
+    metadata:
+        title: tomato
+        description: |
+            tomato: au-tomation without the pain. Instrument automation library. Part of the dgbowl suite of tools for digital (electro-)catalysis and battery materials research.
+        authors: Peter Kraus
+        documentation_url: https://dgbowl.github.com/tomato
```

Comment:
I get a 404 trying to access this page.

Reply:
Yes, that's correct - I didn't get around to deploying the documentation for tomato yet, as we're still fairly far from `1.0`. I can remove the link for now if you'd prefer, but it's where the docs will eventually be, so I thought there's no harm adding it in one PR.

Reply:
Please point to the README file instead in this case and just update the link later. We want to avoid having dead links in the registry.

Looking at the repository README file, I would suggest to just spend 20 minutes to compile some very basic documentation on how to install and use the tool (a single example would be better than nothing), otherwise it is very difficult to nearly impossible for the community to use it. Such basic documentation is also important to be able to understand what the tool does.

Reply:
@PeterKraus Any chance that you could apply this change request as well?

Reply:
Yes, I'm on it. I think I'll have it done next week.

Reply:
Please just ping me here once you're done. Thank you.

Reply:
@csadorf: this should now be ready. The documentation is by no means complete, but at least it has a front page with some basic usage and the `autodocs` are built...